includes/class-wc-payments-token-service.php

```diff
@@ -44,26 +46,33 @@ public function __construct( WC_Payments_API_Client $payments_api_client, WC_Pay
 		add_action( 'woocommerce_payment_token_deleted', [ $this, 'woocommerce_payment_token_deleted' ], 10, 2 );
 		add_action( 'woocommerce_payment_token_set_default', [ $this, 'woocommerce_payment_token_set_default' ], 10, 2 );
 		add_filter( 'woocommerce_get_customer_payment_tokens', [ $this, 'woocommerce_get_customer_payment_tokens' ], 10, 3 );
+		add_filter( 'woocommerce_payment_methods_list_item', [ $this, 'get_account_saved_payment_methods_list_item_sepa' ], 10, 2 );
 	}
 
 	/**
 	 * Creates and add a token to an user, based on the payment_method object
 	 *
 	 * @param array   $payment_method Payment method to be added.
 	 * @param WP_User $user           User to attach payment method to.
-	 * @return WC_Payment_Token_CC    The WC object for the payment token.
+	 * @return WC_Payment_Token_CC|WC_Payment_Token_Sepa    The WC object for the payment token.
 	 */
 	public function add_token_to_user( $payment_method, $user ) {
 		// Clear cached payment methods.
 		$this->customer_service->clear_cached_payment_methods_for_user( $user->ID );
 
-		$token = new WC_Payment_Token_CC();
+		if ( Payment_Method::SEPA === $payment_method['type'] ) {
+			$token = new WC_Payment_Token_Sepa();
+			$token->set_gateway_id( Sepa::GATEWAY_ID );
+			$token->set_last4( $payment_method[ Payment_Method::SEPA ]['last4'] );
+		} else {
+			$token = new WC_Payment_Token_CC();
+			$token->set_gateway_id( WC_Payment_Gateway_WCPay::GATEWAY_ID );
```

Comment:
`Card::GATEWAY_ID` would be a great fit here.

Reply:
I like the idea of using `Card:: GATEWAY_ID`!

Reply:
I see other functions in this file using `WC_Payment_Gateway_WCPay::GATEWAY_ID`. I am tempted to update them to `Card::GATEWAY_ID` but I think the name "WC_Payment_Gateway_WCPay::GATEWAY_ID" makes more sense in those function's context. Although functionality would be the same, I didn't update them. Please let me know if you prefer that done in this PR as well.

Reply:
Thanks for updating the constant reference! Looks good, lets keep other methods without the changes.